tests/test_tron_tools.py

```diff
@@ -1430,7 +1535,7 @@ def test_create_complete_config_e2e(self, tmpdir):
         # that are not static, this will cause continuous reconfiguration, which
         # will add significant load to the Tron API, which happened in DAR-1461.
         # but if this is intended, just change the hash.
-        assert hasher.hexdigest() == "f740410f7ae2794f9924121c1115e15d"
+        assert hasher.hexdigest() == "35972651618a848ac6bf7947245dbaea"
```

Comment:
Note that this may cause a big bounce with all tron jobs when deployed. Make sure you sync with #compute-infra before shipping these changes in case any special considerations are needed to accommodate that.

Reply:
this *should* be fine - the main purpose of this test is too help us catch things like a timestamp (or something equally ever-changing) being added to the podspec as repeatedly reconfiguring every job is quite painful :)

Reply:
luisp++ ah right there in the comment above :) thanks for the context!